Amethyst/Model/Window.swift

```diff
@@ -165,7 +165,7 @@ extension AXWindow: WindowType {
             return false
         }
 
-        return isEqual(to: focused)
+        return focused.windowID() == self.windowID()
```

Comment:
Oh, this is interesting. I wonder if there are other bugs stemming from `isEqual` in `SIAccessibilityElement` doing a direct ref comparison. Either way, though, it is probably better to override `isEqual` in `AXWindow` to make sure the behavior is uniform across other comparisons.